web/sections/canvas-preview/index.tsx

```diff
@@ -1,40 +1,46 @@
 import React, { useState } from "react";
```

Comment:
@You-J  this part conflicts your working area. since @junhoyeo requests PR first, i'll accept this

Reply:
Please note that I just added the name for the default function for more efficient fast refreshing :fire:
The rest hasn’t changed much, except for the linting part lol

> Anonymous arrow functions cause Fast Refresh to not preserve local component state.

Reply:
I understand